### PR TITLE
Update k8s version and add rancher to build

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Story: [AB#7746](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7746)
Task: [AB#10281](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10281)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,28 +5,17 @@ Repository for Platform Terraform code
...
 
-The goal of this is that a branch of the repo would create a dynamic environment for testing Terraform in and confirming the changes have no knock on effect on the application. Once tested the environment can then be removed and the changes will be propegated into Development -> Test -> Production. 
-
-# Modules:
-1) Generate the core key vault with permenant values
-2) Generate the specific key vault for the branch name (environment variable)
-3) Apply the Terraform using the above 2 repos based on branch name (environment variable)
-4) Setup the Kubernetes cluster to work with code
-5) Deploy the code base
+The goal is that all Infrastructure changes can be tested and then  be propegated into Development -> Test -> Production.
```
Comment (bjss-chris) [inline README.md]: Double space between 'then' and 'be'?

File: terraform/application_gateways_dev.tf [hunk 2]
```diff
@@ -120,27 +120,60 @@ resource "azurerm_application_gateway" "AppGate" {
...
+    name                = "rancher"
+    host                = "rancher-${data.azurerm_key_vault_secret.coreurl.value}"
+    interval            = "30"
+    timeout             = "30"
+    unhealthy_threshold = "3"
+    path                = "/"
+    protocol            = "Http"
+  }
+
+
```
Comment (bjss-chris) [inline terraform/application_gateways_dev.tf]: Extra blank line?

File: ingress/helm-agic-config-template.txt [hunk 3]
```diff
@@ -27,8 +27,8 @@ appgw:
 # Leaving this variable out or setting it to blank or empty string would
 # result in Ingress Controller observing all acessible namespaces.
 #
-kubernetes:
-    watchNamespace: ${namespaces}
+#kubernetes:
+#    watchNamespace: ${namespaces}
```
Comment (bjss-chris) [inline ingress/helm-agic-config-template.txt]: Is the plan to reinstate these commented-out lines at some point?

File: ingress/helm-agic-config-template.txt [hunk 4]
```diff
@@ -51,5 +51,5 @@ rbac:
     enabled: true # true/false
 
 # Specify aks cluster related information. THIS IS BEING DEPRECATED.
-aksClusterConfiguration:
-    apiServerAddress: ${apiAddress}
+#aksClusterConfiguration:
+#    apiServerAddress: ${apiAddress}
```
Comment (bjss-chris) [inline ingress/helm-agic-config-template.txt]: Is the plan to reinstate these commented-out lines at some point?

**Review by bjss-chris — COMMENTED** [review]
Just some fairly noddy questions around formatting and commented-out lines.

File: README.md (hunk 1 above)
Comment (gavinwakeley1) [reply README.md]: Removed space

File: terraform/application_gateways_dev.tf (hunk 2 above)
Comment (gavinwakeley1) [reply terraform/application_gateways_dev.tf]: Removed blank line

File: ingress/helm-agic-config-template.txt (hunk 3 above)
Comment (gavinwakeley1) [reply ingress/helm-agic-config-template.txt]: Nope, I'll properly comment them and remove the namespace sed functionality from azure-pipelines. Have left the default template intact (but hashed out) as per AGIC install recommendations.

File: ingress/helm-agic-config-template.txt (hunk 4 above)
Comment (gavinwakeley1) [reply ingress/helm-agic-config-template.txt]: Stripped out of template - old config

**Review by bjss-chris — APPROVED** [review]
Worth @samMcCarthyBJSS taking a quick look with his TF experience?

**Review by samMcCarthyBJSS — APPROVED** [review]

**Pull request merged**